src/Illuminate/Database/Eloquent/Relations/Concerns/InteractsWithPivotTable.php

```diff
@@ -184,6 +184,19 @@ protected function attachNew(array $records, array $current, $touch = true)
      */
     public function updateExistingPivot($id, array $attributes, $touch = true)
     {
+        if ($this->using) {
+            $updated = $this->newPivot([
+                $this->foreignPivotKey => $this->parent->getKey(),
+                $this->relatedPivotKey => $this->parseId($id),
+            ], true)->fill($attributes)->save();
```

Comment:
Only on the third read I realized the second arg to `newPivot` (the boolean).

I'm suggesting something like this:
```php
$updated = $this
    ->newPivot(
        [
            $this->foreignPivotKey => $this->parent->getKey(),
            $this->relatedPivotKey => $this->parseId($id),
        ],
        true
    )
    ->fill($attributes)
    ->save();
```
Yes, more lines, but you harder to "miss" anything.

Reply:
I'm trying to match the style of the framework here. I'm actually indifferent to how it looks, typically @taylorotwell Taylorifies my PRs before final merge, which I'm completely fine with. :)